rcgen/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 
 # Changes
 
-## Unreleased
+## Release 0.12.0 - December 16, 2023
```

Comment:
I think this is missing https://github.com/rustls/rcgen/pull/197 contributed by Brocar.

Proposed text:

- Add `KeyIdMethod::PreSpecified` variant to allow specifying an exact subject key identifier (SKI) instead of a method for generating it based on the subject key. `CertificateParams::from_ca_cert_pem` has been updated to use the pre-existing SKI when building params from an existing CA cert.